pilot_experiment/3_explore_composition_of_corr_genes.ipynb

```diff
@@ -0,0 +1,1473 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Explore composition of correlated genes\n",
+    "\n",
+    "This notebook explores deeper into the genes that are found based on their correlation score."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/home/alexandra/anaconda3/envs/core_acc/lib/python3.7/site-packages/matplotlib/__init__.py:886: MatplotlibDeprecationWarning: \n",
```

Comment:
Any idea which import this warning is coming from? Maybe updating `matplotlib` would get rid of it? Unless it's coming from the `matplotlib_venn` import, in which case it's probably fine to ignore it.

Reply:
Looks like it might be more of a feature than a bug: https://github.com/matplotlib/matplotlib/issues/15781

Reply:
Got it. This is probably fine to ignore then (although you could probably catch the warning when you import `matplotlib` if you want).